src/components/common/styled-components.js

```diff
@@ -374,8 +374,12 @@ export const StyledModalContent = styled.div`
  * This workaround will hide the error banner.
  */
 export const StyledMapContainer = styled.div`
-  .mapboxgl-map .mapboxgl-missing-css {
-    display: none;
+  .mapboxgl-map {
+    z-index: -1;
```

Comment:
why `z-index: -1;` here?

Reply:
this is due to an issue with deckgl where deckgl capture all mouse events despite of other existing layers and their z-index value. So I have to make sure deckgl and its container mapbox-gl has -1 when in editor mode in order to be able to use the mouse to draw polygons.

When in regular mode we have the following:
mapboxgl: -1
deckgl: 0
editor: -1

In editor mode:
mapboxgl: -1
deckgl: -1
editor: 0